README.md

```diff
@@ -147,7 +147,7 @@ for other operating systems. These packages are not guaranteed to be up-to-date.
         * `zypper install micro-editor` (SUSE)
         * `eopkg install micro` (Solus).
         * `pacstall -I micro` (Pacstall).
-        * `su - -c "apt-get install micro"` (ALT)
+        * `apt-get install micro` (ALT Linux)
```

Comment:
Can you please squash your second into your first commit to keep the history clean?

Reply:
> Can you please squash your second into your first commit to keep the history clean?

And how to do it?

Reply:
Is everything right now?